app/models/upload.rb

```diff
@@ -51,6 +55,14 @@ def access_control_post
 
   scope :by_users, -> { where("uploads.id > ?", SEEDED_ID_THRESHOLD) }
 
+  def self.verification_statuses
+    @verification_statuses ||= Enum.new(
+      unchecked: 1,
+      verified: 2,
+      invalid_etag: 3
+    )
+  end
```

Comment:
I am curious if here wouldn't be beneficial to use rails enums.

That would be default define scopes so for example later instead of 
`Upload.where(verification_status: Upload.verification_statuses[:invalid_etag])`
We would be able to simplify it to
`Upload.invalid_etag`

I grepped and we are not using enums too often so maybe there is a reason for that. I only found one in `ApplicationRequest` model

Reply:
I am not sure, we have lib/enum.rb so I am just being consistent. IMO we could easily make the API easier so at least we could do `Upload.verification_statuses.invalid_etag`